internal/pkg/errors/errors.go

```diff
@@ -37,14 +38,14 @@ For more details run:
 
 	DSA_INVALID_INPUT_PLAN = `the instance plan was not correctly provided. 
```

Comment:
The output will be something like:
![image](https://github.com/stackitcloud/stackit-cli/assets/28832884/e6cb7583-a585-47cd-a104-5e130bead013)